plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/FileSystemFinalizerService.java

```diff
@@ -80,6 +80,7 @@ private void processFinalizerQueue()
                 finalizer.cleanup();
             }
             catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
                 return;
```

Comment:
throw?

Reply:
We don’t need to do anything here. This is our own thread. Interrupt means it should exit.